src/strategies/node.ts

```diff
@@ -68,6 +71,29 @@ export class Node extends BaseStrategy {
       }),
     });
 
+    // If a machine readable changelog.json exists update it:
+    if (options.commits && this.pkgJsonContents?.parsedContent) {
+      const pkg = JSON.parse(this.pkgJsonContents.parsedContent);
```

Comment:
the node strategy already has a helper for getting the package name: `getDefaultPackageName()`.